packages/system/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.23.2"
```

```suggestion
- version: "1.24.0"
```

I think this is more adding features in backwards compatible manner than bugfix, so should probably bump the minor version number instead of the patch version number.